cumulus_etl/etl/tasks/base.py

```diff
@@ -0,0 +1,254 @@
+"""ETL tasks"""
```

Comment:
I split up the single tasks.py into a submodule, because it was getting too large.

Unfortunately, along with that refactor, I'm also changing some of the code which makes reviewing a little hard. I could have maybe split them into different commits or PRs to be kind, but I was lazy and here we are.

All the changes here are around supporting multiple output tables (so some interesting changes to the batching iteration utility methods to handle tuples of simultaneous output streams, some changes to the base class EtlTask run support to hand those output streams to multiple Format instances, etc).

The other big change in this folder is the rewrite of MedicationRequestTask to use that new support and download Medications on the fly. So if you want to focus on that rather than the refactor, go look at just that class.